ssv/attester.go

```diff
@@ -227,6 +230,10 @@ func (r *AttesterRunner) GetSigner() types.KeyManager {
 	return r.signer
 }
 
+func (r *AttesterRunner) GetOperatorSigner() types.SSVOperatorSigner {
+	return r.operatorSigner
+}
+
```

Comment:
does runner use the signer?
in the end we use the network to sign it

Reply:
Indeed. Removing it. Thanks!